### ISSUE TITLE
Allow relative or absolute paths for ignore files

### ISSUE THREAD

**Issue by Ikutania** [obicons/sa4u#23]
Currently ignore files require an absolute path to the file. It should allow relative paths from the workspace directory as well.

**Comment by obicons** [comment]
I don’t think this is true. Please be aware the image on docker hub is not necessarily up to date. I think today we match by filename, and ignore the path.

**Comment by obicons** [comment]
You may try pulling the latest code, building a new Docker image, and then running like this (inside of `demos/01`):
```
docker container run  -t -v "$(pwd)":/src/ sa4u-z3 -c /src/compile_commands_dir/ -m /src/CMASI.xml -p /src/ex_prior.json  -i ex.cpp
```

You should see a message like this:
```
INFO: Skipping /src/ex.cpp because it is to be ignored
```

**Comment by obicons** [comment]
@Ikutania how does [this PR](https://github.com/Ikutania/sa4u/commit/9653b5dfe9486f5ba1aa37dc9de47c9cceb0662a) fix this issue?
